docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md

```diff
@@ -1,8 +1,8 @@
 ---
-id: sql-server-opentelemetry
-title: Microsoft SQL Server - OpenTelemetry Collector
-sidebar_label: Microsoft SQL Server - OTel Collector
-description: Learn about the Sumo Logic OpenTelemetry App for Microsoft SQL Server.
+id: sql-server-linux-opentelemetry
```

Comment:
@sumoanema Because you changed this file name, you'll also need to add a 301 redirect to [cid-redirects.json](https://github.com/SumoLogic/sumologic-documentation/blob/main/cid-redirects.json). Otherwise customers (and search engines) will get a 404.